invenio/base/config.py

```diff
@@ -75,24 +75,29 @@
     'invenio.legacy.websubmit',
 ]
 
-CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
+_CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
```

Comment:
Cool. I hadn't used `LocalProxy` before.

I will look into modifying `invenio/ext/logging/__init__.py:deprecated` to work with the new deprecation warning classes and then it should be a little more organized:

```python
@deprecated('Use the other variables instead', (2, 2))
@property
def cfg_prefix():
    return distutils.sysconfig.get_config_var("prefix")
CFG_PREFIX = LocalProxy(cfg_prefix)